rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -367,6 +375,20 @@ private void savePendingFederation(ActivationConfig.ForBlock activations) {
         bridgeStorageAccessor.saveToRepository(PENDING_FEDERATION_KEY.getKey(), serializedPendingFederation);
     }
 
+    private void saveProposedFederation(ActivationConfig.ForBlock activations) {
+        if (!activations.isActive(RSKIP419)) {
+            return;
+        }
+
+        if (!isProposedFederationSet) {
```

Comment:
```suggestion
        if (!activations.isActive(RSKIP419) || !isProposedFederationSet) {
            return;
        }
```

Reply:
Or move it a private method if you prefer

Reply:
Done